fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -239,13 +304,40 @@ class DemographicParity(UtilityParity):
     The :meth:`signed_weights` method will compute the costs according
     to Example 3 of
     `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
+
+    This Moment also supports control features, which can be used to
+    stratify the data, with the Demographic Parity applied within
+    each stratum, but not between strata. If the control feature groups
+    are :math:`c \in \mathcal{C}` then the above equation will become
+
+    .. math::
+      P[h(X) = 1 | A = a, C = c] = P[h(X) = 1 | C = c] \; \forall a, c
     """
 
     short_name = "DemographicParity"
 
-    def load_data(self, X, y, **kwargs):
-        """Load the specified data into the object."""
-        super().load_data(X, y, event=_ALL, **kwargs)
+    def compute_base_event(self, X: pd.DataFrame, y: pd.Series) -> pd.Series:
+        """Compute the base event for demographic parity.
```

Comment:
nit: we have a lot of different ways of capitalizing {D|d}emographic {P|p}arity. Perhaps worth standardizing at some point, not just for DP, but all kinds of others, too.

Reply:
Agreed, but not now.